package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "screwdriver-data-schema",
-  "version": "13.0.0",
+  "version": "12.3.0",
```

Comment:
Well this is going to be awkward. That means that we have to keep track of the fact that version 13 exists, while the `master` branch shows version 12 is the latest.